library/Rules/Attribute.php

```diff
@@ -17,7 +17,13 @@
 use Respect\Validation\Exceptions\ComponentException;
 use Respect\Validation\Validatable;
 
-class Attribute extends AbstractRelated
+/**
+ * Validates an object attribute, event private ones.
+ *
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ * @author Henrique Moody <henriquemoody@gmail.com>
```

Comment:
You're missing your name here.

Also, could you write the docblocks for the methods in the class?